packages/react/src/utils.ts

```diff
@@ -543,22 +543,59 @@ export const isRoleAssignmentsArray = (value: ColumnValueType): value is RoleAss
   return true;
 };
 
-export type RelatedFieldColumn = {
+export const isRelationshipColumn = (value: string) => {
+  const parts = value.split(".");
+  return parts.length === 2 || parts.length === 4;
+};
+
+export const maybeGetRelationshipFromColumnPath = (value: string) => {
+  const parts = value.split(".");
```

Comment:
I'm not sure this is the best strategy -- is it possible to not split the string but instead pass it as a path directly so that we could do `shop.owner.name` or any other valid gql out of the box?

Reply:
But we have to build the selection map and find the related field based on the string alone. How do we get the related field without splitting the string?

Reply:
I think the selection is always the path objectified + true: 

`shop.owner.name` => `{shop: {owner: {name: true}}}`

Reply:
To me, that's the benefit of including the edges/nodes notation in the field string